packages/twenty-emails/src/emails/send-invite-link.email.tsx

```diff
@@ -20,7 +20,7 @@ type SendInviteLinkEmailProps = {
     firstName: string;
     lastName: string;
   };
-  serverUrl?: string;
+  serverUrl: string;
```

Comment:
Overall, I think we should stop authorizing undefined in our typings unless this is needed. The caller should handle the case. This will reduce code complexity